packages/legacy-events/__tests__/ResponderEventPlugin-test.internal.js

```diff
@@ -313,6 +314,7 @@ const run = function(config, hierarchyConfig, nativeEventConfig) {
   // Trigger the event
   const extractedEvents = ResponderEventPlugin.extractEvents(
     nativeEventConfig.topLevelType,
+    RESPONDER_EVENT_SYSTEM,
```

Comment:
Why are we passing `RESPONDER_EVENT_SYSTEM`? That flag was meant for the new event system.